tests/integration/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM golang:1.16 as builder
+ARG CGO_ENABLED=0
+ARG NRJMX_VERSION=1.5.3
```

Comment:
I'd remove the version here, so it becomes less error prone, CI will fail before using an unexpected version
wdyt?

Reply:
But how do I know the file to download in here? 
ADD http://download.newrelic.com/infrastructure_agent/binaries/linux/noarch/nrjmx_linux_${NRJMX_VERSION}_noarch.tar.gz ./

Reply:
I think @varas means removing the default `1.5.3` value, so we get a 404 (rather than an arbitrarily defined default version) if for some reason that's missing from the CI.

Reply:
Fixed